test/playwright/localTest.ts

```diff
@@ -58,9 +58,23 @@ interface LocalServerConfig {
   env?: Record<string, string>;
   base?: string;
 }
+export async function getTemporaryDir() {
```

Comment:
Do we still need this function? I see it's only being used in a custom server test now, not sure if it could be replaced there too?

Reply:
yes, it's using a random directory that's different from the test directory